tests/security/attributes/run_test.pl

```diff
@@ -59,47 +59,58 @@
   die("ERROR: Too many arguments passed");
 }
 
+sub gov_gen {
+    my $aup = shift;
+    my $ejac = shift;
+    my $dpk = shift;
+    my $lpk = shift;
+    my $rpk = shift;
+
+    system("./gov_gen -aup $aup -ejac $ejac -dpk $dpk -lpk $lpk -rpk $rpk -o governance.xml");
```

Comment:
Use existing Perl support libraries to spawn processes

Reply:
Can you be more specific and/or point to an example?

Reply:
```
    my $test = new PerlDDS::TestFramework();
    $test->process($name, $exe);
    $test->start_process($name);
```
I'm not sure if `TestFramework` is best suited to this, but it's one option.  Another would be using `PerlDDS/Process.pm` which inherits from the corresponding class in ACE.